File: inclinic/m/app/src/main/java/com/ontrack/m/MainActivity2.java
package com.ontrack.m;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.Button;
import android.widget.EditText;
import android.view.View;
import android.widget.Spinner;
import android.widget.TextView;
import android.widget.Toast;

import com.google.android.gms.auth.api.credentials.Credentials;

public class MainActivity2 extends AppCompatActivity {
    Button b1;
    EditText ed1, ed2;
    TextView t1,t2;
    int counter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);


        b1=findViewById(R.id.b1);
        t1=findViewById(R.id.item1);
        t2=findViewById(R.id.signup);

        Spinner spinner = findViewById(R.id.lanSpinner);
        String[] spinnerArray={"English","Hindi"};
        ArrayAdapter<String> spinnerArrayAdapter = new ArrayAdapter<String>
                (this, android.R.layout.simple_spinner_item,
                        spinnerArray); //selected item will look like a spinner set from XML
        spinnerArrayAdapter.setDropDownViewResource(android.R.layout
                .simple_spinner_dropdown_item);
        spinner.setAdapter(spinnerArrayAdapter);



        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if (ed1.getText().toString().equals("") &&
                        ed2.getText().toString().equals("")) {
                    Intent intent = new Intent(MainActivity2.this, reg.class);
                    startActivity(intent);
                }



        }

        });
        t1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity2.this,forgotpassword.class);
                startActivity(intent);
            }


        });

        t2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity2.this,signup.class);
                startActivity(intent);
            }


        });
    };
}

File: datashere/app/src/main/java/com/ontrack/datashere/MainActivity2.kt
package com.ontrack.datashere

import android.content.Context
import android.content.Intent
import android.content.SharedPreferences
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity2 : AppCompatActivity() {
    private var sharedPrefFile = "data"
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        val name = findViewById<TextView>(R.id.nametv)
        val email = findViewById<TextView>(R.id.emailtv)
        val phone = findViewById<TextView>(R.id.phonetv)
        val back = findViewById<Button>(R.id.backbtn)

        val sharedPreferences: SharedPreferences = this.getSharedPreferences(sharedPrefFile,
            Context.MODE_PRIVATE)
        val sharedNameValue = sharedPreferences.getString("name_key", "name")
        val sharedNameValue1 = sharedPreferences.getString("email_key", "email")
        val sharedNameValue2 = sharedPreferences.getString("phone_key", "phone")
        name.setText(sharedNameValue)
        email.setText(sharedNameValue1)
        phone.setText(sharedNameValue2)

        back.setOnClickListener(View.OnClickListener {
           finishAffinity()
        })
    }
}